apps/frontend/core-dapp/src/features/trade/open-trade/MarketSlideUp.tsx

```diff
@@ -62,9 +62,7 @@ const MarketSlideUp: React.FC = () => {
         {allPositions
           .filter(
             (position) =>
-              position.market.urlId !== selectedMarket?.urlId &&
-              position.market.resolved !== true &&
-              position.direction !== 'short'
+              position.market.urlId !== selectedMarket?.urlId && position.direction !== 'short'
```

Comment:
```suggestion
              position.market.urlId !== selectedMarket?.urlId && position.direction === 'long'
```
seems less error prone?